src/types.js

```diff
@@ -106,6 +106,8 @@ export const Expectation = {
   CspNotImplemented: "csp-not-implemented",
   CspNotImplementedButReportingEnabled:
     "csp-not-implemented-but-reporting-enabled",
+  CspImplementedButHasDuplicateDirectives:
+    "csp-implemented-but-has-duplicate-directives",
```

Comment:
Nit: Maybe omit the unnecessary `has`?

```suggestion
  CspImplementedButDuplicateDirectives:
    "csp-implemented-but-duplicate-directives",
```